### PR TITLE
add working set trace infrastructure

### PR DESCRIPTION
**Pull request by chang55245**
add working set trace infrastructure

### CODE REVIEW

File: TraceInfrastructure/Backend/BackendTrace.c [hunk 1]
```diff
@@ -168,13 +166,11 @@ void StoreDump(void *address)
...
 
-void DumpStoreAddrValue(void *MemValue, int size)
+void DumpStoreValue(void *MemValue, int size)
 {
     char fin[128];
-    sprintf(fin, "StoreAddress:%#lX\n", (uint64_t)MemValue);
-    WriteStream(fin);
     uint8_t *bitwisePrint = (uint8_t *)MemValue;
-    sprintf(fin, "size:%d, StoreMemValue:", size);
+    sprintf(fin, "StoreMemValue:");
```
Comment (ruhrie) [inline TraceInfrastructure/Backend/BackendTrace.c]: Change to StoreValue

File: TraceInfrastructure/Backend/BackendTrace.c [hunk 2]
```diff
@@ -145,13 +145,11 @@ void LoadDump(void *address)
...
     WriteStream(fin);
 }
-void DumpLoadAddrValue(void *MemValue, int size)
+void DumpLoadValue(void *MemValue, int size)
 {
     char fin[128];
-    sprintf(fin, "LoadAddress:%#lX\n", (uint64_t)MemValue);
-    WriteStream(fin);
     uint8_t *bitwisePrint = (uint8_t *)MemValue;
-    sprintf(fin, "size:%d, LoadMemValue:", size);
```
Comment (ruhrie) [inline TraceInfrastructure/Backend/BackendTrace.c]: Change to LoadValue

**Comment by ruhrie** [comment]
This seems fine mostly. Just two quick gripes. Although my unit tests still pass, you don't have any to test this so I'm going to trust you to verify that everything is still working (although I don't see a reason why it wouldn't).

**Comment by chang55245** [comment]
Yes Rick, I have tested this and printed all keys and values in AtlasUtil/Trace.h, they are good.

File: TraceInfrastructure/Backend/BackendTrace.c [hunk 3]
```diff
@@ -145,13 +145,11 @@ void LoadDump(void *address)
...
+    sprintf(fin, "LoadValue:");
     WriteStream(fin);
     for (int i = 0; i < size; i++)
     {
```
Comment (ruhrie) [inline TraceInfrastructure/Backend/BackendTrace.c]: This still appears to have spaces and not be in hex.

File: TraceInfrastructure/Backend/BackendTrace.c [hunk 4]
```diff
@@ -168,13 +166,11 @@ void StoreDump(void *address)
...
 {
     char fin[128];
-    sprintf(fin, "StoreAddress:%#lX\n", (uint64_t)MemValue);
-    WriteStream(fin);
     uint8_t *bitwisePrint = (uint8_t *)MemValue;
-    sprintf(fin, "size:%d, StoreMemValue:", size);
+    sprintf(fin, "StoreValue:");
     WriteStream(fin);
     for (int i = 0; i < size; i++)
     {
```
Comment (ruhrie) [inline TraceInfrastructure/Backend/BackendTrace.c]: Same with this. Needs to be in hex with no spaces

**Pull request merged**